package.json

```diff
@@ -56,6 +56,7 @@
     "dotenv": "8.2.0",
     "github-oauth": "0.2.2",
     "js-cookie": "2.2.1",
+    "nodemon": "^2.0.4",
```

Comment:
You are right, we do need `nodemon`, thank you for adding this. Can you please remove `^` before the version number to ensure it does not auto update?